modules/apps/src/shared/registry.ts

```diff
@@ -30,3 +31,6 @@ export type AppRegistryInfo = {
 };
 
 export type AppRegistryType = AppRegistryInfo[];
+
+// timeout default values
+export const DEFAULT_APP_TIMEOUT = toBN(10);
```

Comment:
10 blocks is ~2 minutes which is **absolutely** not long enough. Should be at least 24h